src/plugins/intel_gpu/tests/fusions/strided_slice_fusion_test.cpp

```diff
@@ -0,0 +1,80 @@
+// Copyright (C) 2023-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "test_utils.h"
+#include "fusion_test_common.hpp"
+
+#include <intel_gpu/primitives/input_layout.hpp>
+#include <intel_gpu/primitives/data.hpp>
+#include <intel_gpu/primitives/strided_slice.hpp>
+
+#include <cmath>
+
+using namespace cldnn;
+using namespace ::tests;
+
+namespace {
+struct strided_slice_test_params {
+    tensor input_size;
+    tensor output_size;
+    data_types input_type;
+    format input_format;
+    size_t expected_fused_primitives;
+    size_t expected_not_fused_primitives;
+};
+
+class StridedSliceFusingsTest : public ::BaseFusingTest<strided_slice_test_params> {
+public:
+    void execute(strided_slice_test_params& p) {
+        auto input_prim = get_mem(get_input_layout(p));
+
+        network network_not_fused(this->engine, this->topology_non_fused, cfg_not_fused);
+        network network_fused(this->engine, this->topology_fused, cfg_fused);
+
+        network_fused.set_input_data("input", input_prim);
+        network_not_fused.set_input_data("input", input_prim);
+
+        compare(network_not_fused, network_fused, p);
+    }
+
+    layout get_input_layout(strided_slice_test_params& p) {
+        return layout{ p.input_type, p.input_format, p.input_size };
+    }
+
+    format get_input_format(strided_slice_test_params &p) {
+        return p.input_format;
+    }
+};
+
+}  // namespace
+
+/* ----------------------------------------------------------------------------------------------------- */
+/* -------------------------------- StridedSlice cases ------------------------------------------------- */
+/* ----------------------------------------------------------------------------------------------------- */
+
+#define CASE_STRIDED_SLICE_F16_1 { 1, 8, 1, 1 },  { 1, 3, 2, 2 }, data_types::f16, format::bfyx
+
+
+class strided_slice_act_act : public StridedSliceFusingsTest {};
+TEST_P(strided_slice_act_act, basic) {
+    std::vector<int64_t> begin_data = { 0, 0, 0, 0 };
+    std::vector<int64_t> end_data = { 1, 8, 1, 1 };
+    std::vector<int64_t> strides_data = { 1, 1, 1, 1 };
+
+    auto p = GetParam();
+    create_topologies(
+        input_layout("input", get_input_layout(p)),
+        strided_slice("strided_slice", input_info("input"), begin_data, end_data, strides_data, {}, {}, {}, {}, {}, { 1, 8, 1, 1 }),
+        activation("actv_clamp", input_info("strided_slice"), activation_func::clamp),
+        activation("actv_exp", input_info("actv_clamp"), activation_func::exp),
```

Comment:
As of now, it works only because of backward compatibility feature with legacy Activation fusion mechanism. Could you please add proper strided_slice fusion support in prepare_primitive_fusing.cpp?

Reply:
I'm not sure quantize/eltwise could be fused. This test case is made for my issue.
So, I'll add an activation fusion case to prepare_primitive_fusing.cpp.